ci-operator/step-registry/openshift-qe/control-plane/openshift-qe-control-plane-chain.yaml

```diff
@@ -3,6 +3,7 @@ chain:
   as: openshift-qe-control-plane
   steps:
   - chain: openshift-qe-cluster-density-v2
+  - chain: openshift-qe-crd-scale
```

Comment:
I am not sure if we want to keep it as a part of this chain or separate.

Reply:
IMHO it makes the most sense to have in our control-plane work flow to encourage better utilization and $ spend

Reply:
make sense @vishnuchalla ?

Reply:
Yes.